src/components/categories.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/Categories/categories';
 
-const Categories = () => (
-  <div>
-    <button type="button" className="category-btn">Check status</button>
-  </div>
-
-);
+const Categories = () => {
+  const categories = useSelector((state) => state.categories);
+  const dispatch = useDispatch();
+  return (
+    <div>
+      <p>{categories}</p>
```

Comment:
- [ ] Also, we are getting error form there

![Screenshot 2023-05-16 120340](https://github.com/Daniel-Kigozi/react-book2/assets/57408419/25540e9b-5f49-4f08-8910-9923c8f5a74e)

- [ ] Remember that `categories` is an array, We map over the "categories" array and render each category as a `<div>`. Kindly make sure to adjust this code to match your specific object structure and rendering requirements. Check the code below, after, you may delete the ` console.log(categories)` line 👍 

````

const Categories = () => {
  const categories = useSelector((state) => state.categories.categories);
  console.log(categories)
  const dispatch = useDispatch();
  return (
    <div>
      {categories.map((item) => (<div>{item}</div>))}
      <button type="button" onClick={() => dispatch(checkStatus())}>Check status</button>
    </div>
  );
};

````